system-tests/_support/index.js

```diff
@@ -160,6 +160,30 @@ Cypress.addChildCommand("contents", function(elements) {
 });
 
 /**
+ * Sets given JSONString as value for ACE Editor
+ *
+ * This function will set the given text as value for ACE Editor
+ *
+ * @param {jQuery.Element} elements - The DOM scope to search within
+ * @param {String} JSONString - JSON Code as String
+ */
+Cypress.addChildCommand("setJSON", function(elements, JSONString) {
+  if (elements.length != null && JSONString) {
+    elements.map(function(index, element) {
```

Comment:
You aren't modifying the elements, should use `jQuery.each()` instead (and remove the return also).